### ISSUE TITLE
Setup:di:compile fails

### ISSUE THREAD

**Issue by ghost** [EmicoEcommerce/Magento2Tweakwise-archived#3]
Issue Brief
--
When running php bin/magento setup:di:compile or php bin/magento mode:set production the code crashes due to RuntimeException is already in use.

Environment
--
Magento Version: 2.1.5
Tweakwise Version: dev-master
Tweakwise Export Version: dev-master
Environment: (production)
Third party modules: -

Steps to reproduce
--
1. Install Magento 2.1.5
2. Install Tweakwise via composer
3. Run php bin/magento setup:di:compile

Actual result
--
1. php bin/magento setup:di:compile
Compilation was started.
Repositories code generation... 1/7 [====>-----------------------]  14% 1 sec 46.0 MiB
PHP Fatal error:  Cannot use RuntimeException as RuntimeException because the name is already in use in /var/www/techtwo/m2-clients/johnbeerens/johnbeerens.com/build/revision-e420b7557dadb6a8c3a641496d98179e3c53697e/vendor/emico/tweakwise/src/Exception/ApiException.php on line 11

Expected result
--
- setup:di:compile runs smoothly without errors

**Comment by ghost** [comment]
Possible solution would be:

Change code in vendor/emico/tweakwise/src/Exception/ApiException.php to:

```
<?php
/**
 * Tweakwise & Emico (https://www.tweakwise.com/ & https://www.emico.nl/) - All Rights Reserved
 *
 * @copyright Copyright (c) 2017-2017 Tweakwise.com B.V. (https://www.tweakwise.com)
 * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
 */

namespace Emico\Tweakwise\Exception;

class ApiException extends \Emico\Tweakwise\Exception\RuntimeException implements TweakwiseException
{

}
```

**Comment by martijnrmediact** [comment]
We are having the same issue here.

**Comment by Fgruntjes** [comment]
Resolved with c2d3871152028a96a9b9746fbfb0160789a35fdb
